controllers/operations/opsrequest_controller.go

```diff
@@ -540,6 +546,30 @@ func (r *OpsRequestReconciler) deleteCreatedPodsInKBNamespace(reqCtx intctrlutil
 	return nil
 }
 
+func (r *OpsRequestReconciler) deleteParametersWithOwner(reqCtx intctrlutil.RequestCtx, opsRequest *opsv1alpha1.OpsRequest) error {
+	listOpts := []client.ListOption{
+		client.InNamespace(opsRequest.GetNamespace()),
+		client.MatchingLabels{
+			constant.OpsRequestNameLabelKey: opsRequest.Name,
+			constant.AppInstanceLabelKey:    opsRequest.Spec.ClusterName,
+		},
+	}
+	parameterCRs := &parametersv1alpha1.ParameterList{}
+	if err := r.Client.List(reqCtx.Ctx, parameterCRs, listOpts...); err != nil {
+		return err
+	}
+	for i := range parameterCRs.Items {
+		parameter := &parameterCRs.Items[i]
+		if !model.IsOwnerOf(opsRequest, parameter) {
```

Comment:
If an OwnerReference is already set on these Parameter CRs, then when the OpsRequest CR is deleted, they will be automatically deleted by the cascading deletion mechanism, so there is no need to explicitly delete them.